spec/menu_spec.rb

```diff
@@ -0,0 +1,52 @@
+require './lib/menu'
+
+describe Menu do
+
+  describe '#menu' do
+    it "defines the method" do
+      expect(subject).to respond_to(:menu)
+    end
+
+    it "stores the items and prices in array of hashes" do
+      expect(subject.menu).to eq [
+        { item: "Noodles", price: 6.00 },
+        { item: "Prawn Cocktail", price: 3.80 },
+        { item: "Prawn Crackers", price: 1.80 },
+        { item: "Deep Fried Seaweed", price: 4.00 }
+      ]
+    end
+  end
+
+  describe '#show_menu' do
+    it "defines the method" do
+      expect(subject).to respond_to(:show_menu)
+    end
+
+    it "formats the menu" do
+      allow(subject.show_menu).to receive(:show_menu) { "#{i + 1}. #{k[:item]}: £#{k[:price]}" }
```

Comment:
This test doesn't have an expectation, so isn't currently testing anything. Also, try to avoid stubbing methods on the subject currently under test.